File: 백준/Bronze/3052. 나머지/나머지.java

import java.util.HashSet;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        HashSet<Integer> g = new HashSet<Integer>();

        for(int i=0; i<10; i++) {
            g.add(sc.nextInt() % 42);
        }
            System.out.println(g.size());
                }
            }

File: 백준/Bronze/10430. 나머지/나머지.kt
import java.util.*

fun main() = with(Scanner(System.`in`)) {
    var A = nextInt()
    var B = nextInt()
    var C = nextInt()
    
    println((A+B)%C)
    println((A%C + B%C)%C)
    println(A*B%C)
    println(A%C * (B%C)%C)
}

